project-folder/CNN.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>can we make sure that all of these libraries are used? zipfile, cv2 and loadtxt are not being referenced anywhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>